src/main/java/frc/robot/Robot.java

```diff
@@ -114,7 +115,8 @@ public Robot() {
 			swerve.getConstants().kinematics(),
 			swerve.getModules().getWheelsPositions(0),
 			swerve.getAbsoluteHeading(),
-			PoseEstimatorConstants.DEFAULT_ODOMETRY_STANDARD_DEVIATIONS
+			PoseEstimatorConstants.DEFAULT_ODOMETRY_STANDARD_DEVIATIONS,
+			new VisionDenoiser(20)
```

Comment:
constants?

Reply:
I added a constant in the robot template. 
@YHerm you can preform a merge between two different project, consider doing that